frontend/src/components/molecules/DetailView.tsx

```diff
@@ -39,8 +46,15 @@ const DetailView: React.FC = () => {
     setDevelopmentAreas(await getDevelopmentArea(selectedNode.id));
   };
 
-  const expandConnection = (connection: Node) => {
+  const loadObjectPropertyDefinition = async () => {
+    if (!selectedPredicate) return;
+    setObjectAnnotations(await getAnnotations(selectedPredicate[1]));
+  };
```

Comment:
The naming scheme here is a little confusing as the the name of the function is `loadObjectPropertyDefinition`, but everything it does seems to be related to annotations. I might be a little out of the loop, but it seems as though annotations are different from ObjectPropertyDefinitions